File: AuctionWebApp.Server/Data/Dto/SimulationInfo.cs
namespace AuctionWebApp.Server.Data.Dto
{
    public class SimulationInfo
    {
        public ulong InitialPrice { get; set; }
        public ulong PriceStep { get; set; }
        public int CyclesCount { get; set; }
        public byte AuctionTypeId { get; set; }
        public IEnumerable<SimulationUser> Users { get; set; } = new List<SimulationUser>();
    }
}

File: auctionwebapp.client/src/app/simulationInfo.ts
import { SimulationUser } from "./simulationUser";

export class SimulationInfo {
  constructor(
    public InitialPrice: number = 0,
    public PriceStep: number = 0,
    public CyclesCount: number = 1,
    public AuctionTypeId: string = "1",
    public Users: SimulationUser[] = []
  ) { }
}
